go.mod

```diff
@@ -3,24 +3,16 @@ module github.com/kubeflow/common
 go 1.16
 
 require (
+	github.com/go-logr/logr v0.4.0
 	github.com/go-openapi/spec v0.20.3
-	github.com/prometheus/client_golang v1.10.0
-	github.com/sirupsen/logrus v1.6.0
-	github.com/stretchr/testify v1.6.1
-	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
-	golang.org/x/sys v0.0.0-20210510120138-977fb7262007 // indirect
-	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	k8s.io/api v0.19.9
-	k8s.io/apimachinery v0.19.9
-	k8s.io/client-go v0.19.9
-	k8s.io/code-generator v0.19.9
-	k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
+	github.com/prometheus/client_golang v1.11.0
+	github.com/sirupsen/logrus v1.7.0
+	github.com/stretchr/testify v1.7.0
+	k8s.io/api v0.21.3
+	k8s.io/apimachinery v0.21.3
```

Comment:
One more request, could we split the dependency upgrade PR to a separate one?

Reply:
sure.